open-hackathon-adminUI/src/app/admin/login.py

```diff
@@ -0,0 +1,179 @@
+__author__ = 'root'
+
+import sys
+import urllib2
+sys.path.append("..")
+# -*- coding:utf8 -*-
+# encoding = utf-8
+from app.functions import get_remote, get_config, post_to_remote, convert
+from app.log import log
+import json
+from admin_mgr import admin_manager
+
+
+class LoginBase():
+
+    def login2db(self,openid,**kwargs):
+
+        admin_with_token = admin_manager.db_login(openid, **kwargs)
+        # login flask
+        admin = admin_with_token["admin"]
+        log.info("github user login successfully:" + repr(admin))
```

Comment:
change the logs
Login class tells which provider ; 
BaseLogin only tells the login status : success